kobo/apps/hook/views/v2/hook_log.py

```diff
@@ -78,6 +84,10 @@ def get_queryset(self):
         # Django 1.9+, "select_related() prohibits non-relational fields for
         # nested relations."
         queryset = queryset.select_related('hook__asset')
+
+        status = self.request.GET.get('status')
+        if status is not None:
+          queryset = queryset.filter(status=status)
```

Comment:
You should add some validations on the `status` field and return a ValidationError in case user does not pass a valid status name. 

If you use `STATUSES_MAPPING`, you'd better move it to `constants.py` if your validation takes place in the serializer.

Reply:
OK the latest commit will throw on in valid statuses - but still expect the numeric status id...